src/Card/CardFlipper.component.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from "react"
+import styled, { css } from "styled-components"
+import { CardProps } from "./Card.component"
+
+export interface CardFlipperProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Nice job on making sure we support all valid HTML attributes for this component!